recipes-apps/ovmenu-ng/files/openvario-57-lvds/ovmenu-ng.sh

```diff
@@ -50,6 +50,7 @@ function submenu_file() {
 	--begin 3 4 \
 	--menu "You can use the UP/DOWN arrow keys" 15 50 4 \
 	Download_IGC   "Download XCSoar IGC files to USB" \
+	Delete_IGC   "Deletes saved IGC files" \
```

Comment:
Do you still want to include igc wiping with this feature? To my understanding, file selection should remove the need for it.

Reply:
Let's remove it. With file selection this is not as important now.